api/app/signals/apps/zds/workflow.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+
+from signals.apps.signals.workflow import (
+    AFGEHANDELD,
+    AFGEHANDELD_EXTERN,
+    AFWACHTING,
+    BEHANDELING,
+    GEANNULEERD,
+    GEMELD,
+    LEEG,
+    ON_HOLD,
+    TE_VERZENDEN,
+    VERZENDEN_MISLUKT,
+    VERZONDEN
+)
+
+# This is a wrapper for the ZTC statusses.
+# This needs to be filled with links to the ZTC status.
+BASE_URL = '{}/statustypen/'.format(settings.ZTC_ZAAKTYPE_URL)
+
+ZTC_STATUSSES = {
```

Comment:
hardcoded?

Reply:
Created an issue. For now searching is not possible in the API.

https://github.com/VNG-Realisatie/gemma-zaken/issues/642